### DISCUSSION TITLE
useTransition abrupt transition between mount on and off

### DISCUSSION THREAD

**Discussion by branlok** [pmndrs/react-spring#1576] [Support]
On leave it seems to just get cut off abruptly... it is suppose to do that?
Im trying to get mount in fade in and mount off fade out animation.

recorded from https://react-spring.io/hooks/use-transition#mountunmount-single-component-reveals  below

https://gyazo.com/08ce4c04b188d6e0f7a665252dce24c0

**Comment by branlok** [comment]
it looks like if taken delay out - it returns to the expected behaviour.

**Reply by camurai** [reply]
Ah thanks branlok! I was running into this same issue and pulling my hair out. This is definitely a bug as the examples on the useTransition documentation have the same issue of not running the out transitions. its pretty obvious on the emoji toggle example. I will post it as an issue if it is not already there.

**Reply by camurai** [reply]
I did not find an existing issue report for this so I posted one: https://github.com/pmndrs/react-spring/issues/1662
